unlock_indicator.c

```diff
@@ -261,7 +301,7 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     timeinfo = localtime(&rawtime);
 
     if (unlock_indicator &&
-        (unlock_state >= STATE_KEY_PRESSED || auth_state > STATE_AUTH_IDLE || show_clock)) {
```

Comment:
I was thinking of something like that, yeah. I'll be testing/poking at it for the next couple hours!